### DISCUSSION TITLE
Welcome to doctorpatientportal Discussions!

### DISCUSSION THREAD

**Discussion by thegr8dev** [thegr8dev/doctorpatientportal#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sandeepsoni2291** [comment]
Actually I am trying to register Doctor but its throwing some error can you please suggest me the solution of this error

Warning: Undefined array key "psques" in C:\xampp\htdocs\doctor-portal-live\dp.php on line 23

Warning: Undefined array key "answer" in C:\xampp\htdocs\doctor-portal-live\dp.php on line 24

**Reply by thegr8dev** [reply]
Hey,

Actually this may be cause of the php version make sure you're using php 5.6

**Reply by bishalmindset** [reply]
Hi there, i am having same issue. Are you planning to update your code? specially dp.php file

**Reply by bishalmindset** [reply]
> Actually I am trying to register Doctor but its throwing some error can you please suggest me the solution of this error
> 
> Warning: Undefined array key "psques" in C:\xampp\htdocs\doctor-portal-live\dp.php on line 23
> 
> Warning: Undefined array key "answer" in C:\xampp\htdocs\doctor-portal-live\dp.php on line 24

Hi Sandeep
Have you able to fix?

**Reply by thegr8dev** [reply]
HI,

Currently I'm planning to upgrade this project to Laravel version so actual payment gateway can also be implemented with more new features and UI.

Till that there is only work around that you need to install php 5.6

**Comment by bishalmindset** [comment]
I see
When will you release new version?

**Comment by amjadameur** [comment]
Hi Ankit, I added support for Docker and Vagrant to your project for automization purposes: https://github.com/amjadameur/doctor_webapp
Regards,
Amjad

**Reply by thegr8dev** [reply]
Hey,

Thanks for making it.
